[2201.7.0] Update release note with few OpenAPI changes (#7200)
ballerina-platform/ballerina-dev-website: downloads/swan-lake-release-notes/swan-lake-2201.7.0/RELEASE_NOTE.md

- Added a new API named `workspaceProjects` to the workspace manager to get the projects loaded in the workspace.

#### OpenAPI Tool

> - Added a new flag, `--without-data-binding`, to the OpenAPI to Ballerina service generation command for generating low-level service skeletons without any data-binding logic. This flag can be used for implementing pass-through services.

Reviewer: ```suggestion
- Added a new `--without-data-binding` flag to the OpenAPI to Ballerina service generation command for generating low-level service skeletons without any data-binding logic. This flag can be used for implementing pass-through services.
```

- Added a new API named `workspaceProjects` to the workspace manager to get the projects loaded in the workspace.

#### OpenAPI Tool

> - Added support to generate low-level service skeletons without any data-binding logic. This mode can be enabled by adding `--without-data-binding` flag to the OpenAPI to Ballerina service generation command. This flag can be used for implementing pass-through services.

Reviewer: ```suggestion
- Added support to generate low-level service skeletons without any data-binding logic. This mode can be enabled by adding the `--without-data-binding` flag to the OpenAPI to Ballerina service generation command. This flag can be used for implementing pass-through services.
```

#### OpenAPI tool
- Added auto-generated file headers for all the generated Ballerina files in OpenAPI to Ballerina service and client generations. Also, the users will have the option to  replace the auto generated headers with their own license, using the `--license <license-file-path>` command option.

> - Added support for query parameters with object type in the OpenAPI to Ballerina service generation.

Reviewer: ```suggestion
- Added support for query parameters with an object type in the OpenAPI to Ballerina service generation.
```

> - Added support for query parameters with an object type in the OpenAPI to Ballerina service generation.

Reviewer 2: ```suggestion
- Added support for query parameters with object types in the OpenAPI to Ballerina service generation.
```
